packages/wellbeing-mock-data/wellbeing-mock-data.js

```diff
@@ -130,20 +130,106 @@ function createMockActivities() {
   }
 }
 
+
+function getRandomHomeButExcludeCurrent(current_id) {
+  var homeIds = Homes.find().map(function (e) { return e._id; });
+  // remove current home from array
+  var indexOfCurrent = homeIds.indexOf(current_id);
+  if (indexOfCurrent > -1) {
+    homeIds.splice(indexOfCurrent, 1);
+  }
+  //return random element from rest of the homes
+  return homeIds[Math.floor(Math.random() * homeIds.length)];
+}
+  /*
+    Get random date from the range of today - startingpoint
+    @param start = months from today
+    creates two dates one of them being today
+    and second one being today minus months set in parameters
+  */
+function getRandomMoveInDate(start) {
+  var endDate = new Date();
+  var startDate = new Date();
+  startDate.setMonth(endDate.getMonth() - start);
+  moveInDate = randomDate(endDate, startDate);
+  return moveInDate;
+}
+
+  /*
+    This function assumes residents are already added.
+    then adds residency for every resident
+    Depending on the percentMovedOut some residents will have multiple residencies
+
+  */
+function createMockResidency(startingPoint, percentMovedOut) {
+  console.log("Creating Mock Residencies")
+  //get all residents
+  var residents = Residents.find().fetch();
+
+  /*  this is the number from where residents are moved out. So if the percentMovedout is 0.25
+      this index will be at the 75% percent point of the resident list
+      when the loop reaches this index it will move the residents randomly either to another home or just move them out completely
+  */
+  var indexWhereMovedOut = residents.length - Math.round(residents.length * percentMovedOut)
+  for (var i = 0; i < residents.length; i++) {
```

Comment:
What about using [array.forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) here instead if a `for` loop?